### ISSUE TITLE
The longest journey - April is stuck with the synthetic eye

### ISSUE THREAD

**Issue by carpedee** [residualvm/residualvm#1506]
Hello, I just wanted to report a bug I encountered with the last daily build for ResidualVm (November 17) : when April is in the police station, she normally can replace the detective' synthetic eye by the monkey toy one, enabling her to bypass a retinal scan later. But here I don't have that possibility : when selecting the monkey toy's eye it never flickers when putting it on top of the detective's eye, meaning it's impossible to swap them and to go further in the game. There is also a weird flickering happening on the toilets stalls at the same moment.
I tried the different shaders and the fullscreen/windowed mode but it didn't change anything.
I joined the save corresponding to this situation.
[tlj-win-001.zip](https://github.com/residualvm/residualvm/files/2597338/tlj-win-001.zip)

**Comment by bgK** [comment]
> April is in the police station, she normally can replace the detective' synthetic eye by the monkey toy one. But here I don't have that possibility : when selecting the monkey toy's eye it never flickers when putting it on top of the detective's eye,

Indeed, there is an issue with the synthetic eye, but actually the hotspot is only badly placed. You can still perform the exchange by placing the mouse a bit lower and to the right.

**Comment by carpedee** [comment]
> > April is in the police station, she normally can replace the detective' synthetic eye by the monkey toy one. But here I don't have that possibility : when selecting the monkey toy's eye it never flickers when putting it on top of the detective's eye,
> 
> Indeed, there is an issue with the synthetic eye, but actually the hotspot is only badly placed. You can still perform the exchange by placing the mouse a bit lower and to the right.

Thanks for the tip, it worked. But this misplaced hitbox is really troublesome, I spent two days missing the exact spot
